ecs/disks.go

```diff
@@ -90,7 +90,10 @@ type DescribeDisksResponse struct {
 }
 
 // DescribeDisks describes Disks
+//
+// You can read doc at:http://docs.aliyun.com/#/pub/ecs/open-api/disk&describedisks
 func (client *Client) DescribeDisks(args *DescribeDisksArgs) (disks []DiskItemType, pagination *PaginationResult, err error) {
+
```

Comment:
Oh, my mistake! 
I wrote "fwrite.WriteString("\n")"  in my program addlink. I will recorrect it  and then update yufeizyf/aliyungo。